testing/src/test/java/org/motechproject/nms/testing/it/kilkari/CsrServiceBundleIT.java

```diff
@@ -467,7 +467,62 @@ public void verifyFT149() {
                 subscriptionPackDataService, languageDataService, languageLocationDataService, circleDataService,
                 stateDataService, districtDataService);
 
-        Subscription subscription = sh.mksub(SubscriptionOrigin.MCTS_IMPORT, DateTime.now());
+        Subscription subscription = sh.mksub(SubscriptionOrigin.MCTS_IMPORT, DateTime.now(),SubscriptionPackType.CHILD);
+        String contentFileName = sh.getContentMessageFile(subscription, 0);
+
+        csrDataService.create(new CallSummaryRecord(
+                new RequestId(subscription.getSubscriptionId(), "11112233445566").toString(),
+                subscription.getSubscriber().getCallingNumber(),
+                "w1_1.wav",
+                "w1_1",
+                makeLanguageLocation().getCode(),
+                makeCircle().getName(),
+                FinalCallStatus.FAILED,
+                makeStatsMap(StatusCode.OBD_FAILED_BUSY, 1),
+                0,
+                10,
+                3
+        ));
+
+        Map<Integer, Integer> callStats = new HashMap<>();
+        callStats.put(StatusCode.OBD_FAILED_BUSY.getValue(),1);
+        CallSummaryRecordDto record = new CallSummaryRecordDto(
+                new RequestId(subscription.getSubscriptionId(), timestamp),
+                subscription.getSubscriber().getCallingNumber(),
+                contentFileName,
+                "XXX",
+                makeLanguageLocation().getCode(),
+                makeCircle().getName(),
+                FinalCallStatus.FAILED,
+                callStats,
+                0,
+                3
+        );
+
+        Map<String, Object> eventParams = new HashMap<>();
+        eventParams.put(CSR_PARAM_KEY, record);
+        MotechEvent motechEvent = new MotechEvent(PROCESS_SUMMARY_RECORD_SUBJECT, eventParams);
+        csrService.processCallSummaryRecord(motechEvent);
+
+        CallRetry callRetry = callRetryDataService.findBySubscriptionId(subscription.getSubscriptionId());
+        assertNotNull(callRetry);
+        assertTrue(CallStage.RETRY_1.equals(callRetry.getCallStage()));
+    }
+
+    @Test
+    public void verifyFT141() {
+
+        String timestamp = DateTime.now().toString(TIME_FORMATTER);
+
+        // To check that NMS shall retry OBD message for which delivery fails for the first time with two message per
```

Comment:
Looks like a copy/paste error, this code marks the failure as DND so this won't test the right thing and verifyFT149 (which should test DND) marks the calls as FAILED instead of REJECTED.